include/sound/sof/ipc4/header.h

```diff
@@ -423,6 +423,12 @@ enum sof_ipc4_fw_config_params {
 	SOF_IPC4_FW_CFG_RESERVED,
 	SOF_IPC4_FW_CFG_POWER_GATING_POLICY,
 	SOF_IPC4_FW_CFG_ASSERT_MODE,
+	SOF_IPC4_TELEMETRY_BUFFER_SIZE,
+	SOF_IPC4_BUS_HARDWARE_ID,
+	SOF_IPC4_DEBUG_WINDOW_SIZE,
+	SOF_IPC4_KPB_RT_SINK_COUNT,
+	SOF_IPC4_DMI_FORCE_L1_EXIT,
```

Comment:
do we need to expose all of the new definitions? Should we have a comment that says which ones are actually used by the driver? I don't know e.g. what the "BUS_HARDWARE_ID" might refer to.

Reply:
These config id are used by ref fw. I will change them to SOF_IPC4_RESERVED_xxx